src/rpc/rawtransaction.cpp

```diff
@@ -415,6 +417,18 @@ UniValue createrawtransaction(const UniValue &params, bool fHelp)
 
         uint32_t nSequence =
             (rawTx.nLockTime ? std::numeric_limits<uint32_t>::max() - 1 : std::numeric_limits<uint32_t>::max());
+
+        // set the sequence number if passed in the parameters object
+        const UniValue &sequenceObj = find_value(o, "sequence");
+        if (sequenceObj.isNum())
+        {
+            int64_t seqNr64 = sequenceObj.get_int64();
+            if (seqNr64 < 0 || seqNr64 > std::numeric_limits<uint32_t>::max())
+                throw JSONRPCError(RPC_INVALID_PARAMETER, "Invalid parameter, sequence number is out of range");
+            else
+                nSequence = (uint32_t)seqNr64;
+        }
+
```

Comment:
if its not a number we should throw a JSONRPCError, not just silently ignore it.

Reply:
Yeah you have point.

Reply:
On second thought why not just rising an error if `sequence` is not a number?

Reply:
My point was what if someone used "25" as the sequence, not 25 -- that is, for some json translation reason the number is being sent as a string.  There is a tension in the RPC interface between end-user (command-line and scripts) convenience and programmatic correctness (where really we can insist that 25 be 25 not "25".  Since end-user convenience doesn't much impact programmatic use case (except possibly by hiding bugs), I tend to favor convenience.

But we aren't going to solve this today,  For example, the RPC interface only wants 1 or 0 for true/false values which is pretty awkward.  So I am ok with this being a rigid, integer-only parameter.